src/status_im/bots/events.cljs

```diff
@@ -59,6 +50,28 @@
   [{:keys [current-chat-id] :as app-db}]
   (assoc-in app-db [:bot-db current-chat-id] nil))
 
+(def ^:private keywordize-vector (partial mapv keyword))
+
+;; TODO(janherich): do something with this horrible triple nested reduce so it's more readable
+(defn add-active-bot-subscriptions
```

Comment:
Please introduce smaller functions. 

Also add tests to make sure we don't introduce regressions.

Reply:
Technically we can't introduce regression to functionality which is totally broken and not working and all (as are bot subscriptions currently in develop) :) 
But I get your point, will introduce smaller functions and some basic tests.

Reply:
Let's do this in separate PR